core/src/main/java/io/lionweb/lioncore/java/language/LionCoreBuiltins.java

```diff
@@ -38,30 +47,61 @@ private LionCoreBuiltins() {
   }
 
   public static LionCoreBuiltins getInstance() {
```

Comment:
What about declaring this method `@Deprecated`? It's almost certainly wrong to use it.

Same for all the other version-less methods.

Reply:
I think most users would use the variants that are version-less, because they want to use the latest version and when they migrate to a new version this would be seemless. I think very few users will have a reason to use a specific version and not just the latest one